docs/Learning-Environment-Examples.md

```diff
@@ -81,6 +82,7 @@ If you would like to contribute environments, please see our
     * Vector Action space: (Continuous) Size of 2, corresponding to movement toward net or away from net, and jumping.
     * Visual Observations: None
 * Reset Parameters: One, corresponding to size of ball.
+* Benchmark Total Cumulative Reward: 0
```

Comment:
Is it really 0 ?

Reply:
my understanding this would eventually move toward 0, since losing / winning would correspond to negative / positive.  let me know if i am mistaken

Reply:
thanks for clarifying.  running and will add the benchmark in